src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateManager.cs

```diff
@@ -186,6 +199,16 @@ public static X509Certificate2 CreateAndInstallLocalMachineCertificates(Certific
                 InstallCertificateToRootStore(rootCertificate);
                 InstallCertificateToMyStore(hostCert, certificateCreationSettings.ValidityType == CertificateValidityType.Valid);
                 s_localCertificate = hostCert;
+
+                // Create the PeerTrust cert
+                certificateCreationSettings = new CertificateCreationSettings()
+                {
+                    FriendlyName = "WCF Bridge - UserPeerTrustCertificateResource",
+                    Subject = fqdn,
+                    SubjectAlternativeNames = new string[] { fqdn, hostname, "localhost" }
+                };
+                var peerCert = certificateGenerator.CreateMachineCertificate(certificateCreationSettings).Certificate;
+                InstallCertificateToTrustedPeopleStore(peerCert, certificateCreationSettings.ValidityType == CertificateValidityType.Valid);
             }
```

Comment:
We may need to just create a new self-signed cert that doesn't chain up to the CA cert in any way. Will think on how to achieve this